acceptance-tests.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+docker run -d -h oauth --name oauth -t danpersa/rusty-oauth:latest
+docker run -d -h postgres -e POSTGRES_PASSWORD= -e POSTGRES_USER=innkeepertest --name db -t postgres:9.4
+sbt  assembly
```

Comment:
why do you start postgres and oauth before building the assembly? and why are there two spaces in-between `sbt` and `assembly`?